SPECS/dietlibc/CVE-2015-1473.patch

```diff
@@ -0,0 +1,56 @@
+From 3e76f9036b2845c988969cfeaefa6b5bd236185f Mon Sep 17 00:00:00 2001
+From: jykanase <v-jykanase@microsoft.com>
+Date: Mon, 17 Feb 2025 10:11:29 +0000
+Subject: [PATCH] CVE-2015-1473
+
+Source Link: https://sourceware.org/git/?p=glibc.git;a=patch;h=5bd80bfe9ca0d955bfbbc002781bc7b01b6bcb06
```

Comment:
The original patch fixes two files along with some documentation. Among the two files only `tst-sscanf.c` is present in this package's source tar. So, importing fixes for only that file.

Though it is a test-only fix, there is no harm in adding the fix.